spark-connector/spark-common/src/test/java/org/apache/gravitino/spark/connector/integration/test/sql/SparkTestConfig.java

```diff
@@ -0,0 +1,123 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.gravitino.spark.connector.integration.test.sql;
+
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import org.apache.gravitino.Config;
+import org.apache.gravitino.config.ConfigBuilder;
+import org.apache.gravitino.config.ConfigEntry;
+import org.junit.platform.commons.util.StringUtils;
+
+public class SparkTestConfig extends Config {
+  private static final String DEFAULT_BASE_DIR =
+      Paths.get(
+              System.getenv("GRAVITINO_ROOT_DIR"),
+              "spark-connector",
+              "spark-common",
+              "src",
+              "test",
+              "resources")
+          .toString();
+
+  private static final ConfigEntry<String> TEST_BASE_DIR =
+      new ConfigBuilder("gravitino.test.sql.dir")
+          .doc("The Spark SQL test base dir")
+          .version("0.6.0")
+          .stringConf()
+          .createWithDefault(DEFAULT_BASE_DIR);
+
+  private static final ConfigEntry<String> TEST_SQLS =
+      new ConfigBuilder("gravitino.test.sqls")
+          .doc(
+              "Specify the test SQLs, using directory to specify group of SQLs like `test-sqls/hive`, using file path to specify one SQL like `test-sqls/hive/basic.sql`, use `,` to split multi part")
```

Comment:
This is too long, we should limit the length into 100 chars.

Reply:
done